docs/API.md

```diff
@@ -604,10 +608,14 @@ immediately upon experiencing a "gap" in sequences.
 | `SRTO_MAXBW`          | 1.0.5 | pre     | `int64_t` | bytes/sec | -1       | -1     |
 
 - **[GET or SET]** - Maximum send bandwidth.
-- `-1`: infinite (CSRTCC limit is 30mbps)
-- `= 0`: relative to input rate (SRT 1.0.5 addition, see `SRTO_INPUTBW`) 
-- `>0`: absolute limit 
-- *SRT recommended value: 0 (relative)*
+- `-1`: infinite (the limit in Live Mode is 1Gbps)
+- `0`: relative to input rate (see `SRTO_INPUTBW`) 
+- `>0`: absolute limit in B/s
+
+- *NOTE: This option has a default value of -1. For cases when the stream rate
+is mostly constant however, it is recommended to use value 0 here and shape the
+bandwidth limit using `SRTO_INPUTBW` and `SRTO_OHEADBW` options.*
```

Comment:
I think "however" should be in the beginning of the sentence. @stevomatthews

Reply:
I think this is correct.

Reply:
How about we drop the "however" entirely?
"This option has a default value of -1. For cases when the stream rate is mostly constant, it is recommended to set the value to 0, and to shape the bandwidth limit using the `SRTO_INPUTBW` and `SRTO_OHEADBW` options."

Reply:
"However" is here exactly because the recommended value 0 in given cases contradicts to the default value -1.

Reply:
I can change it to "though", could be smarter :)